components/concept/engine/src/main/java/mozilla/components/concept/engine/Engine.kt

```diff
@@ -162,6 +163,31 @@ interface Engine {
         )
     )
 
+    /**
+     * Provides access to the tracking protection exception list for this engine.
+     */
+    val trackingProtectionExceptionStore: TrackingProtectionExceptionStorage
+        get() =
```

Comment:
I think we should just let the getter throw if the store is not supported by this engine, rather than every method separately. Probably not likely we will ever have partial store support? And even if we do, we can throw on individual methods later. For now this could just be:

```kotlin
val trackingProtectionExceptionStore: TrackingProtectionExceptionStorage
        get() = throw UnsupportedOperationException("TrackingProtectionExceptionStorage not supported by this engine.")
```

Reply:
Sounds great 👍